### ISSUE TITLE
$x .= m.n is a trap

### ISSUE THREAD

**Issue by zopsicle** [Raku/doc#3333]
## The problem

The following two lines are equivalent:

```raku
$x .= split(',').grep(?*);
($x .= split(',')).grep(?*);
```

The following two lines are not:

```raku
$x .= split(',').grep(?*);
$x .= split(','); $x .= grep(?*);
```

Although the latter is probably what somebody expects.

Thanks to @AlexDaniel for clarifying this in IRC and finding at least two examples of the trapped:

 * https://github.com/moritz/perl6-all-modules/blob/08c1a2471fcb8508881862f88cb348c464745f42/cpan/ELIZABETH/List-MoreUtils/lib/List/MoreUtils.pm6#L335
 * https://github.com/moritz/perl6-all-modules/blob/08c1a2471fcb8508881862f88cb348c464745f42/cpan/HYTHM/Grid/lib/Grid.pm#L398

## Suggestions

Document this as a trap in the page that lists traps.

**Comment by 2colours** [comment]
See also: https://github.com/rakudo/rakudo/issues/4863

**Comment by ab5tract** [comment]
I believe this addressed successfully in 992d269.
